src/components/ArtworkItem.jsx

```diff
@@ -10,11 +10,11 @@ const ArtworkItem = ({ data }) => (
     <img
       className="w-full h-64 py-4 px-4"
       src={data?.images?.web ? `${data.images?.web?.url}` : Desing1}
-      alt="producto"
+      alt={data.title ? data.title : "Item"}
```

Comment:
What do you think about use:
```suggestion
      alt={data.title ? data.title : "Nombre de la obra"}
```

?